authorize_request.go

```diff
@@ -25,12 +25,22 @@ import (
 	"net/url"
 )
 
+type ResponseModeType string
+
+const (
+	ResponseModeNone     = ResponseModeType("")
```

Comment:
Rename this `ResponseModeDefault`. Because there is also `ResponseModeNone`: https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html#none

Reply:
And also implement it please. :-)

Reply:
+1 for renaming it to `ResponseModeDefault`
Regarding supporting `none`.  @mitar Isn't the link about `response_type` not the `response_mode`?

Reply:
Oh, you are right. Sorry, I got confused here.